nodejs/eks/package.json

```diff
@@ -23,8 +23,8 @@
     "bugs": "https://github.com/pulumi/pulumi-eks/issues",
     "dependencies": {
         "@pulumi/aws": "6.65.0",
-        "@pulumi/kubernetes": "4.18.3",
-        "@pulumi/pulumi": "^3.143.0",
```

Comment:
Wondering if this will introduce issues similar to https://github.com/pulumi/pulumi-aws-apigateway/pull/178 that necessitates keeping a range or using a special YARN syntax?